recipes/mpiplus/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "mpiplus" %}
+{% set version = "v0.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/choderalab/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: a3d50157a274decab87589ad78b9e41e9f3d34eb27a7d50cc6e7ef34def98b44 
+
+build:
+  number: 0
+  skip: True  # [py<35]
```

Comment:
I recommend againts using the `py` selector since there have been some historical bugs with it.  Instead, given current conda-forge builds you can do `[py27]`, since we are no longer building python 3.4